bindings/node/lib/index.d.ts

```diff
@@ -109,6 +109,16 @@ export declare interface AccountToCreate {
   createdAt?: string;
 }
 
+export declare enum StorageType {
+  Stronghold = 1,
```

Comment:
Any reason not to start at 0 :) ?

Reply:
no reason really, I could change it later (there's another enum starting at 1)